### PR TITLE
Refactored Directory Browser Taks.

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**

### CODE REVIEW

**Comment by mgenov** [comment]
The only problem are the hardcoded values for "Dir:" and "File: ". 

This could easily be solved using enum or simple boolean variable in the FileBox class: 
boolean directory

So the callers will be able to: 

```java
if (fileBox.isDirectory()) {
   System.out.println(String.format(fileBox.file().Name() + " is directory."));
}
```

**Comment by AlexanderVladimirov9090** [comment]
Something like this?

File: dirbrowser/src/FileBox.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import java.io.File;
+
+/**
+ * Created by clouway on 02.06.16.
+ */
+public class FileBox {
+  private boolean isDir;
```
Comment (mgenov) [inline dirbrowser/src/FileBox.java]: "is" is not required on the field, cause Getter for the boolean "dir" field is already isDir.

**Comment by mgenov** [comment]
Looks much better now.

**Pull request merged**